pkg/monitortests/testframework/metricsendpointdown/monitortest.go

```diff
@@ -0,0 +1,94 @@
+package metricsendpointdown
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/monitortests/clusterversionoperator/operatorstateanalyzer"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+	"k8s.io/client-go/rest"
+)
+
+const testName = "[sig-node] kubelet metrics endpoints should not be unreachable"
```

Comment:
'should not be unreachable' -> 'should be reachable' might be a little more readable.

Reply:
changed to "should always be reachable"